pitch.md

```diff
@@ -57,13 +98,17 @@ the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+About 60% of the time will be spent thinking about the langauge aspects. There are keywords and syntax that need to be decided, though the implementation of each of the language features should not be _too_ complex. A large part of the time will be spend building out the right mechanisms to ensure that user queries are parsed appropriately. (And that user queries are intuitive to read and write!)
```

Comment:
I think that one could potentially sink a lot of time into the system that provides the language. It would be good to put bounds on how much you're going to work on the system.